src/calibration.c

```diff
@@ -37,3 +37,18 @@ void run_angular_speed_profile(void)
 	set_target_linear_speed(0.);
 	each(10, log_angular_speed, 2000);
 }
+
+/**
+ * @brief Run profiling test for sensors distance calculation.
+ */
```

Comment:
I would add a more detailed description to specify that this basically executes 1000 times the sensor readings and then logs the time it took for all the executions to complete.

Reply:
Detailed description added, for me was clear..., typical error, :)